pages/index.js

```diff
@@ -52,13 +67,40 @@ const StyledNav = styled.nav`
   line-height: 1.2rem;
 `;
 
+const StyledBackButton = styled.input`
+  background-color: transparent;
+  border: none;
+  text-decoration: none;
+  color: black;
+  margin: 1rem;
+  padding: 1rem;
+  border-radius: 8.5px;
+  border: 1px solid black;
+  text-align: center;
+  background-color: lightskyblue;
+`;
```

Comment:
Redundant CSS, `border: none` is overwritten by `border: 1px solid black`
```suggestion
const StyledBackButton = styled.input`
  background-color: transparent;
  text-decoration: none;
  color: black;
  margin: 1rem;
  padding: 1rem;
  border-radius: 8.5px;
  border: 1px solid black;
  text-align: center;
  background-color: lightskyblue;
`;
```